installer

```diff
@@ -0,0 +1,139 @@
+#!/usr/local/cpython-3.4/bin/python3
+
+"""
+Copy modules from here to --prefix.
+
+Note that we do not scan for directories; instead we have a hardcoded list in all_directories.
+"""
+
+import os
+import shutil
+import sys
+
+
+prefix = os.path.expanduser('~/.micropython/lib')
+directories = []
+all_directories = ['micropython', 'python-ecosys', 'python-stdlib', 'tools', 'unix-ffi']
+
+
+def usage(retval):
+    """Output a usage message."""
+    if retval == 0:
+        write = sys.stdout.write
+    else:
+        write = sys.stderr.write
+
+    write('Usage: {} --prefix ~/.micropython/lib --directories {}\n'.format(sys.argv[0], ' '.join(all_directories)))
+    write('--directories, if provided, must be the last argument specified\n')
+
+    sys.exit(retval)
+
+
+# The getopt-like module du jour would be a little more concise, but that's much harder for a static analyzer to check.
```

Comment:
argparse is hardly 'du jour' and arguably the statis analyzer is not good if it cannot handle that

Reply:
If you know of a static analyzer that does this, I'll probably want to switch.

Reply:
What needs to be checked, and what problem are you having with argparse?

Reply:
Code needs to be checked, and I haven't used an argument parser since getopt.

Reply:
Can you be more concrete? I.e. this comment sounds like something doesn't work for a particular static analyzer. As such it raises more questions that it answers. My question really is: what does not work, and for which analyzer.